getting-started/templates/systemlink-values.yaml

```diff
@@ -987,6 +1014,133 @@ fileingestion:
+    ## The Elasticsearch host. If using the SystemLink Elasticsearch deployment
+    ## provide the FQDN of the headless service
+    host: "elasticsearch-release-master-hl.default.svc.cluster.local"
+
+  ## MongoDB configuration
+  ##
+  mongodb:
+    ## The name of the secret to be created containing the MongoDB password.
+    ## This secret will be created by the deployment. Password needs to be provided in the secrets section.
+    secretName: "fileingestioncdc-mongodb-credentials"
+    ## The MongoDB connection string override, use this to explicitly configure or override the global configuration for MongoDB
+    connection_string: ""
+    ## The MongoDB database
+    database: "files"
+    ## The MongoDB user
+    username: "filescdc"
+
+  ## High Availability configuration
+  ##
+  highAvailability:
+    ## Whether to enable high availability. This is required for production environments.
+    ## Needs an object storage medium, like S3, Azure Storage or a ReadWriteMany volume.
+    enabled: true
+    ## Storage configuration
+    ##
+    storage:
+      ## The type of the object storage medium. Supported values are "s3", "azure" and "readWriteManyVolume"
+      type: "s3"
+      ## Configure S3/MinIO access.
+      s3:
+        ## Secret name for S3 credentials
+        secretName: "flink-aws-keys"
+        ## S3 access type. Supported values are "ACCESS_KEY" and "AWS_WEB_IDENTITY_TOKEN".
+        authType: "ACCESS_KEY"
+        ## The name of the S3 bucket that the service should connect to.
+        bucket: "systemlink-flink"
+        ## Set this value to connect to an external S3 instance.
+        host: "s3.amazonaws.com"
+        ## Set this value to connect to an S3 instance which is internal to the cluster.
+        service: ""
+        ## S3 Region.
+        region: "us-east-1"
+        ## The name of the key in the above secret whose value contains the S3 AccessKeyId.
+        accessKeyIdKey: "aws-access-key-id"
+        ## The name of the key in the above secret whose value contains the S3 SecretAccessKey.
+        secretAccessKeyKey: "aws-secret-access-key"
+      ## Configure Azure Storage access.
+      azure:
+        ## The name of the Azure Storage account. Required if Azure Storage is enabled.
+        accountName: ""
+        ## The name of the secret containing the Azure Storage credentials.
+        secretName: "flink-azure-credentials"
+        ## The name of the Azure Storage container that the service should connect to.
+        containerName: "systemlink-flink"
+        ## The name of the key in the above secret whose value contains the shared access key.
+        accessKeyName: "azure-secret-access-key"
+        ## Whether to use TLS for connections to Azure Storage.
+        isSecure: true
+        ## The Azure Blob Storage host and port to connect to, not including the account name.
+        blobApiHost: "blob.core.windows.net"
+      ## Configure ReadWriteManyVolume access.
+      readWriteManyVolume:
```

Comment:
I would also remove this one

Reply:
I'm also not sure if I would remove this, maybe I'll add something in the comments to make it more suggestive that it should'nt be configured only if you select the type of storage, I'll also do that for the other two